app/carbonapi/http_handlers_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"net/http"
 	"testing"
 
-	"github.com/bookingcom/carbonapi/pkg/types"
+	typ "github.com/bookingcom/carbonapi/pkg/types"
```

Comment:
Should we change a reference not within this commit?
I see no conflicts in changes. Let's keep the diff as small as possible.

Reply:
This is an ongoing change. In many places, little by little. This is just one step of many. So, let's keep it.

Reply:
Ok then.